src/r_things.c

```diff
@@ -959,10 +959,6 @@ void R_DrawPlayerSprites(void)
   mfloorclip = screenheightarray;
   mceilingclip = negonearray;
 
-  // display crosshair
-  if (hud_crosshair)
```

Comment:
I think we had crosshair drawing right here for a reason, i.e. to prevent it from drawing on top of the weapon sprite when looking all the way up but aiming straight ahead.

Reply:
![woof0000](https://github.com/user-attachments/assets/06348c19-9363-4153-adc5-1f5e07e36e1e)

Reply:
> to prevent it from drawing on top of the weapon sprite when looking all the way up but aiming straight ahead.

Is this a problem? It's annoying when the crosshairs disappear.

Reply:
But it looks borked when it's drawn over the weapon sprite.

Reply:
Thank you!